config/database.php

```diff
@@ -43,7 +43,7 @@
             'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
         ],
 
-        'mysql' => [
+        'ecommerce' => [
```

Comment:
nao retire o driver mysql
```suggestion
        'mysql' => [
```